xchem_db/models.py

```diff
@@ -21,46 +177,44 @@ class Meta:
         db_table = 'tasks'
         unique_together = ('task_name', 'uuid')
 
-
 class Target(models.Model):
-    target_name = models.CharField(max_length=255, blank=False, null=False, unique=True, db_index=True)
 
+    target_name = models.CharField(max_length=255, blank=False, null=False, unique=True, db_index=True)
     # uniprot_id = models.CharField(blank=True, null=True)
     # alias = models.CharField(blank=True, null=True)
 
     class Meta:
         app_label = 'xchem_db'
         db_table = 'target'
 
-
-class Compounds(models.Model):
-    smiles = models.CharField(max_length=255, blank=True, null=True, db_index=True, unique=True)
-
-    class Meta:
-        if os.getcwd() != '/dls/science/groups/i04-1/software/luigi_pipeline/pipelineDEV':
-            app_label = 'xchem_db'
-        db_table = 'compounds'
-
-
 class Reference(models.Model):
     reference_pdb = models.CharField(max_length=255, null=True, default='not_assigned', unique=True)
 
     class Meta:
         if os.getcwd() != '/dls/science/groups/i04-1/software/luigi_pipeline/pipelineDEV':
-            app_label = 'xchem_db'
+           app_label = 'xchem_db'
         db_table = 'reference'
 
-
+#modified: 'proposal' changed to 'name', added SoakDB-related attributes and __str__
 class Proposals(models.Model):
-    # TODO - can we refactor this for title
-    proposal = models.CharField(max_length=255, blank=False, null=False, unique=True, db_index=True)
+
+    # TODO - can we refactor this for title [original comment]
+    name = models.CharField(max_length=255, blank=False, null=False, unique=True)
```

Comment:
Maybe change back to proposal, I don't know who this affects. We should be able to change pipeline code to accommodate no biggie

Reply:
leave as proposal